portfolio/src/main/java/com/google/sps/data/Task.java

```diff
@@ -13,32 +14,55 @@
   private final Optional<String> description;
   private final Duration duration;
   private final TaskPriority priority;
+  private Instant scheduledTime;
```

Comment:
I don't really like how a `Task` will sometimes have a `scheduledTime` and sometimes not, and there's not really any way to know about it ahead of time. Consider instead creating a wrapper class like:
```
public class ScheduledTask {
  private final Task task;
  private final Instant startTime;

  ...
}
```

then you can statically know whether a `Task` has been scheduled or not.

Reply:
I implemented ScheduledTask.